### PR TITLE
fix: subnav with scrolling sticky table

### PR DESCRIPTION
**Pull request by TorstenDittmann**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/lib/elements/table/cellCheck.svelte [hunk 1]
```diff
@@ -24,7 +24,7 @@
     };
 </script>
 
-<TableCell class="u-position-relative">
+<TableCell>
     <div
         class="touch-area"
```
Comment (ArmanNik) [inline src/lib/elements/table/cellCheck.svelte:29]: This class is not needed anymore?

**Review by ArmanNik — APPROVED** [review]

File: src/lib/elements/table/cellCheck.svelte (hunk 1 above)
Comment (TorstenDittmann) [reply src/lib/elements/table/cellCheck.svelte:29]: this messes up the sticky table cells - I dont think there was ever a need for this class 🤔

Comment (TorstenDittmann) [reply src/lib/elements/table/cellCheck.svelte:29]: it was only needed because the class down below added position absoulute

Comment (ArmanNik) [reply src/lib/elements/table/cellCheck.svelte:29]: can we remove it then?

**Review by ArmanNik — APPROVED** [review]

**Pull request merged**